flocx_market/api.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from flask import Flask, request
+from flask_restful import Api
+from flask_migrate import Migrate
+import pymysql
+from dotenv import load_dotenv
+from db import db
+
+from resources.offer import Offer, OfferList
+
+app = Flask(__name__)
+load_dotenv(".env")
+app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get('DATABASE_URI')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['PROPAGATE_EXCEPTIONS'] = True
+api = Api(app)
+migrate = Migrate(app, db)
+api.add_resource(Offer, '/offer', '/offer/<string:marketplace_offer_id>')
+api.add_resource(OfferList, '/offers')
+
+if __name__ == '__main__':
+    db.init_app(app)
+
+    app.run(port=8080, debug=True)
```

Comment:
I'm guessing this shouldn't be here, as it's cmd/app.py that starts up the app?

Reply:
Yes, Leo and I have moved api.py to cmd as app.py.

Reply:
@DanNiESh if we move this file to cmd/app.py, we should definitely leave the if __name__=='__main__' in there with the db init app and app.run

Reply:
Actually, you shouldn't have the `if name == '__main___'` block in there *anywhere*. You should be setting up a console_scripts entrypoint so that the service can be run from the command line.  See e.g. https://github.com/CCI-MOC/esi-leap/blob/master/setup.cfg#L15

Reply:
okay then we need to take out db.init_app(app) into another file since that can only be called once, otherwise, as I read somewhere some kind of recursive imports will attempt to initialize the ORM every time app is used in another file and this can cause a lot of headaches. 
The name=='__main__' was also restricting it from calling db.init_app(app) when app is imported in another file. 
I'll play around with it.